### DISCUSSION TITLE
Translation/Language of videos

### DISCUSSION THREAD

**Discussion by Alphacharge** [FreeTubeApp/FreeTube#6260] [Q&A]
Hey, i have following situation:
My system is set to german locale, because i need this for an other app. Freetube is set to english, because i want to use it in english.
Now videos are sometimes translated to english. Like i read Youtube does translate specific videos automatically right?
And i guess Freetube is fetching the language i set in the settings.
Is there a way to avoid this? 
I want the videos to stay in the original language created.
In fact it's fetching original german videos in the english version and i have to change it manually back.
Thx

**Comment by absidue** [comment]
Please read my message here: https://github.com/FreeTubeApp/FreeTube/issues/4291#issuecomment-2452527942

**Reply by Alphacharge** [reply]
Ok, thank you. I only searched for translation keywords everywhere.
